### DISCUSSION TITLE
1.3.0什么时候更新？

### DISCUSSION THREAD

**Discussion by keji422** [PenUniverse/PenMods#175] [你问我答]
1.3.0什么时候更新？

**Comment by Redbeanw44602** [comment]
我知道你很急，但你先别急😅

**Reply by Galaxy-YCY** [reply]
跪求支持词典笔3专业版（目前系统版本2.73）

**Comment by Zack88604** [comment]
https://github.com/PenUniverse/PenMods-release/discussions/194#discussioncomment-8715630
